corehq/apps/app_manager/static/app_manager/template_apps/agriculture/app.json

```diff
@@ -1 +1 @@
e('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/rice_yield_weight\" vellum:value=\"#case/rice_yield_weight\" /> kilos</value>\n\t\t\t\t\t\t<value form=\"markdown\">**Rice harvested:** <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/rice_yield_weight\" vellum:value=\"#case/rice_yield_weight\" /> kilos</value>\n\t\t\t\t\t</text>\n\t\t\t\t\t<text id=\"report_group/Rice_report_group_label/label_crop_yield_rice_unknown-label\">\n\t\t\t\t\t\t<value>**Rice harvested to date:** no harvest information recorded for <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/case_name\" vellum:value=\"#case/case_name\" /></value>\n\t\t\t\t\t\t<value form=\"markdown\">**Rice harvested to date:** no harvest information recorded for <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/case_name\" vellum:value=\"#case/case_name\" /></value>\n\t\t\t\t\t</text>\n\t\t\t\t\t<text id=\"report_group/Rice_report_group_label/label_crop_earnings_rice-label\">\n\t\t\t\t\t\t<value>**Earnings from rice harvest:** <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/crop_value_rice\" vellum:value=\"#case/crop_value_rice\" /> dollars</value>\n\t\t\t\t\t\t<value form=\"markdown\">**Earnings from rice harvest:** <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/crop_value_rice\" vellum:value=\"#case/crop_value_rice\" /> dollars</value>\n\t\t\t\t\t</text>\n\t\t\t\t\t<text id=\"report_group/Rice_report_group_label/label_crop_earnings_rice_unknown-label\">\n\t\t\t\t\t\t<value>**Earnings from rice harvest to date:** no price information recorded for <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/case_name\" vellum:value=\"#case/case_name\" /></value>\n\t\t\t\t\t\t<value form=\"markdown\">**Earnings from rice harvest to date:** no price information recorded for <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/case_name\" vellum:value=\"#case/case_name\" /></value>\n\t\t\t\t\t</text>\n\t\t\t\t\t<text id=\"report_group/Corn_report_group_label-label\">\n\t\t\t\t\t\t<value>#### CORN REPORT</value>\n\t\t\t\t\t\t<value form=\"markdown\">#### CORN REPORT</value>\n\t\t\t\t\t</text>\n\t\t\t\t\t<text id=\"report_group/Corn_report_group_label/label_crop_yield_corn-label\">\n\t\t\t\t\t\t<value>**Corn harvested:** <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/corn_yield_weight\" vellum:value=\"#case/corn_yield_weight\" /> bushels</value>\n\t\t\t\t\t\t<value form=\"markdown\">**Corn harvested:** <output value=\"instance('casedb')/casedb/case[@case_id = instance('commcaresession')/session/data/case_id]/corn_yie
```

Comment:
Not blocking: it would be nice to pretty-print these template apps so that in future PRs we could see what changed.